hdf5.spec

```
@@ -1,29 +1,27 @@
 %global macrosdir %(d=%{_rpmconfigdir}/macros.d; [ -d $d ] || d=%{_sysconfdir}/rpm; echo $d)
 %{!?_fmoddir:%global _fmoddir %{_libdir}/gfortran/modules}
 
-%global daos_major 0
-%global hdf5_commit fa40c6c59af5d9aabd4b478cd02f8a9f7ebf7922
-%define hdf5_sha .gfa40c6c59a
+%global daos_major 1
 
 # Patch version?
 %global snaprel %{nil}
-# HDF5 versions
-%global major 1.12
-%global minor 0
+%global hdf5_major 1
+%global hdf5_minor 13_0
+%global hdf5_release rc5
 
 # NOTE:  Try not to release new versions to released versions of Fedora
 # You need to recompile all users of HDF5 for each version change
 Name: hdf5
-Version: %{major}.%{minor}
-Release: 5%{hdf5_sha}%{?dist}
+Version: %{hdf5_major}_%{hdf5_minor}
+Release: %{hdf5_release}%{?commit:.git%{shortcommit}}%{?dist}
```

As mentioned above the `rc5` is part of the upstream version and belongs in `Version:` (prefixed with a `~`).

I do like this idea of making the appending of the commit something that doesn't need changing from iteration to iteration and is done (or not) depending on whether a commit has been defined.

```suggestion
Release: 1%{?commit:.git%{shortcommit}}%{?dist}
```